lib/src/klio/transforms/io.py

```diff
@@ -360,6 +360,19 @@ class KlioReadFromAvro(beam.io.ReadFromAvro):
     Data from avro is dumped into JSON and assigned to ``KlioMessage.data.
     element``.
 
+    ``KlioReadFromAvro`` is the default read for event input config type avro.
+    However,``KlioReadFromAvro`` can also be called explicity in a pipeline.
+
+    Example pipeline reading in elements from an avro file:
+
+    .. code-block:: python
+
+        def run(pipeline, job_config):
```

Comment:
For this one and the write to avro example, can you use "config" instead of "job_config"?